subsys/bluetooth/controller/ll_sw/lll.h

```diff
@@ -315,6 +321,10 @@ enum {
 #endif /* CONFIG_BT_CTLR_ADV_EXT */
 #endif /* CONFIG_BT_OBSERVER */
 
+#if defined(CONFIG_BT_CTLR_CONN_ISO_STREAMS)
```

Comment:
This Kconfig was intended for something else, but it will work as expected and the naming is consistent with how it is used here. I guess we can leave it at this for now, but we should clean up the Kconfigs for a consistent terminology in the future (not in the least to simplify the `PERIPHERAL_ISO || CENTRAL_ISO` guards).

Reply:
We'll leave it like this for now.